code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -255,6 +255,27 @@
 /datum/species/skrell/can_breathe_water()
 	return TRUE
 
+/datum/species/skrell/handle_environment_special(var/mob/living/carbon/human/H)
+	var/old_moisture = H.moisture
+	H.moisture = max(0, H.moisture - rand(-0.8, 1.3))			// Reduce moisture a little.
```

Comment:
Moisture loss should be dependent on air temperature. The hotter air, dryer they get.